src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/FindPaginationTableIntegrationTest.java

```diff
@@ -1,23 +1,30 @@
 package io.stargate.sgv2.jsonapi.api.v1.tables;
 
 import static io.stargate.sgv2.jsonapi.api.v1.util.DataApiCommandSenders.assertTableCommand;
+import static io.stargate.sgv2.jsonapi.api.v1.util.scenarios.TestDataScenario.fieldName;
 
 import io.quarkus.test.common.WithTestResource;
 import io.quarkus.test.junit.QuarkusIntegrationTest;
-import io.stargate.sgv2.jsonapi.api.v1.util.scenarios.KeyValueTable100Scenario;
+import io.stargate.sgv2.jsonapi.api.v1.util.scenarios.PaginationTable100Scenario;
+import io.stargate.sgv2.jsonapi.api.v1.util.scenarios.ThreeClusteringKeysTableScenario;
+import io.stargate.sgv2.jsonapi.exception.RequestException;
 import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
 import java.util.List;
 import java.util.Map;
 import org.junit.jupiter.api.*;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
```

Comment:
This is bad import (probably due to IDEA auto-complete), please fix to use Guava `ImmutableMap`, instead of private shaded copy by testcontainers.